integration-tests/src/main.rs

```diff
@@ -22,9 +22,11 @@ enum Cli {
         nodes: usize,
         #[arg(short, long, default_value_t = 2)]
         threshold: usize,
+        #[arg(long, default_value = "ws://localhost:8545")]
+        eth_rpc_ws_url: String,
         #[arg(long, default_value = "http://localhost:8545")]
-        eth_rpc_url: String,
-        #[arg(long, default_value = "0x5FbDB2315678afecb367f032d93F642f64180aa3")]
+        eth_rpc_http_url: String,
+        #[arg(long, default_value = "99bbA657f2BbC93c02D617f8bA121cB8Fc104Acf")]
```

Comment:
Should we revert this one? Current deploy script is expected to deterministically deploy local contract to 0x5FbDB2315678afecb367f032d93F642f64180aa3